test/variants/ub_rec06.v3

```diff
@@ -0,0 +1,28 @@
+//@execute 0=0; 1=5; 2=10; 3=120
+
+type T<E> #unboxed {
+	case Leaf(x: E);
+	case One(a: T<E>.Leaf, b: T<E>.Leaf);
+	case Two(a: T<E>.One, b: T<E>.One);
+}
+
+def main(a: int) -> int {
```

Comment:
Can you add a third test which has two different instantiations of this polymorphic type, with one instantiation being a tuple or other type that normalizes to more than one scalar?

Reply:
I'll merge this for now; I can do the bump and the test.